integrationtest/src/test/java/org/hibernate/ogm/test/integration/cassandra/ModulesMagicDeckIT.java

```diff
@@ -44,7 +44,7 @@
 				.addClasses( MagicCard.class, MagicCardsCollectionBean.class, ModulesMagicDeckIT.class, MagiccardsDatabaseScenario.class );
 		String persistenceXml = persistenceXml().exportAsString();
 		webArchive.addAsResource( new StringAsset( persistenceXml ), "META-INF/persistence.xml" );
-		ModulesHelper.addModulesDependencyDeclaration( webArchive, "org.hibernate.ogm services, org.hibernate.ogm.cassandra services" );
+		ModulesHelper.addModulesDependencyDeclaration( webArchive, "org.hibernate.ogm:${hibernate-ogm.module.slot} services, org.hibernate.ogm.cassandra:${hibernate-ogm.module.slot} services" );
```

Comment:
Indeed, I did all those changes to make it work in the IDE.

The maven build-helper plugin doesn't work in the IDE, so while we use that to define the major.minor properties, the Integration tests re-compute the values from a different source (and inject these) using explicit Java code, so to keep the ability to run integration tests easily from the IDE.